examples/tree/panel.js

```diff
@@ -61,7 +61,8 @@ Ext.application({
         });
 
         treeStore = Ext.create('GeoExt.data.TreeStore', {
-            layerGroup: olMap.getLayerGroup()
+            layerGroup: olMap.getLayerGroup(),
+            showLayerGroupNode: true
```

Comment:
The "layerGroupNode" is the second node of the tree, so every name containing "root" could be misleading. Inheritance from the panel seems to be usefull, as the visibility of a node is of course a view configuration and not about the data.

I'd like to show this property in the example. If we decide, that this a usecase that never happens we better remove it completly and always hide the layerGroupNode.